encodings/fastlanes/src/bitpacking/compute/search_sorted.rs

```diff
@@ -219,9 +219,12 @@ mod test {
         #[case] side: SearchSortedSide,
         #[case] expected: SearchResult,
     ) {
+        use vortex_array::variants::PrimitiveArrayTrait;
```

Comment:
can we move this to the module level, i.e. outside this function but in `mod tests`